### PR TITLE
Newsletter June 2021

### PR DESCRIPTION
**Pull request by Robadob**
*I'd like to send this out early Monday, but didn't get a chance to finish it off Friday, so a review ASAP would be nice.*

### CODE REVIEW

**Comment by Robadob** [comment]
Will noted that SeptemRSE probably justifies being a news item rather than events.

**Comment by Robadob** [comment]
https://www.eventbrite.co.uk/e/funder-panel-challenges-opportunities-of-funding-research-software-tickets-157219609083?aff=ebdssbonlinesearch

**Comment by Robadob** [comment]
https://twitter.com/RSE_Sheffield/status/1400472923048185858

**Comment by Robadob** [comment]
@willfurnass I noticed that github doesn't resolve the footer link for IT-services correctly (it's in the split link format) https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/blob/master/_newsletters/its-res-it is 404. Not sure if this is a problem, but the other similar links resolve.

**Comment by Robadob** [comment]
Ah fixed it, 2nd set of brackets was `()` instead of `[]`

File: _newsletters/2021-06-07-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+To suggest content for a future newsletter either complete [this short form](https://forms.gle/YyQiNjg79tXaLZHq8) or drop us an [email](mailto://rse@sheffield.ac.uk). We can’t promise that we’ll publish everything that’s proposed but we are keen to feature submissions from the research community at the University of Sheffield.
+
+*All times are BST (UTC+01)*
+
+
+### News 
+
+#### SeptembRSE
+
+The [Conference of Research Software Engineers](https://septembrse.society-rse.org/) is returning online this year.
```
Comment (willfurnass) [inline _newsletters/2021-06-07-newsletter.md]: 'returning online' implies it's been run online previously

**Review by willfurnass — APPROVED** [review]
Looks great.  All links are valid.  One very minor comment/suggestion.

**Pull request merged**